native/cocos/audio/android/AudioEngine-inl.cpp

```diff
@@ -396,3 +397,78 @@ void AudioEngineImpl::onResume() {
         _audioPlayerProvider->resume();
     }
 }
+
+PCMHeader AudioEngineImpl::getPCMHeader(const char* url) {
+    PCMHeader header{};
```

Comment:
I suggest cache procedure is expected here.

Reply:
This problem is already recordede in the issue https://github.com/cocos/cocos-engine/issues/11615. Will be optimized once header is unified